addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -314,10 +317,13 @@ export class LinkPopover extends Component {
     }
 
     get classes() {
+        var classes = [...this.props.linkElement.classList]
+            .filter((value) => value != "btn" && !value.includes("btn-fill-"))
+            .join(" ");
         if (!this.state.type) {
-            return "";
+            return classes;
         }
-        return `btn btn-fill-${this.state.type}`;
+        return (`btn btn-fill-${this.state.type} ` + classes).trim();
```

Comment:
you need to rebase cause of a very recent change to add customized style for buttons :grimacing: